t/api/08-jobtemplates.t

```diff
@@ -832,6 +832,60 @@ subtest 'Create and modify groups with YAML' => sub {
           || diag explain $t->tx->res->body;
     };
 
+    subtest 'Multiple scenarios with different variables' => sub {
+        # Define more than one scenario with the same testsuite
+        my %foo_spam = (
+            settings => {
+                FOO => 'spam',
+            },
+        );
+        my %foo_eggs = (
+            settings => {
+                FOO => 'eggs',
+            },
+        );
+        $yaml->{scenarios}{i586}{'opensuse-13.1-DVD-i586'} = [{foobar => \%foo_spam}, {foobar => \%foo_eggs}];
+        $t->post_ok(
+            "/api/v1/experimental/job_templates_scheduling/$job_group_id3",
+            form => {
+                template => YAML::XS::Dump($yaml)}
+        )->status_is(400, 'Post rejected because scenarios are ambiguous')->json_is(
+            '' => {
+                error => [
+                        'Scenario \'foobar\' is defined more than once.'
+                      . 'Use a unique name and specify \'testsuite\' to re-use test suites with multiple scenarios.'
```

Comment:
These two lines are concatenated without any space in between causing https://travis-ci.org/os-autoinst/openQA/jobs/580683033#L1034 . Better add the space, e.g. after the `.` one line above